Assets/MRTK/SDK/Features/UX/Scripts/Collections/GridObjectCollectionSizeFitter.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities
+{
+    [ExecuteInEditMode]
+    public class GridObjectCollectionSizeFitter : MonoBehaviour
+    {
+        [SerializeField]
+        [Tooltip("The GridObjectColletion to fit")]
+        private GridObjectCollection GridObjectCollection;
+
+        [SerializeField]
+        [Range(0.001f, .5f)]
+        [Tooltip("Scale factor for the border. Stored in local Z scale.")]
+        private float BorderScale = 0.02f;
+
+        [SerializeField]
+        [Range(0, .1f)]
+        [Tooltip("Total margin added to left and right")]
+        private float HorizontalMargin = 0.0f;
+
+        [SerializeField]
+        [Range(0, .1f)]
+        [Tooltip("Total margin added to top and bottom")]
+        private float VerticalMargin = 0.0f;
+
+        [SerializeField]
+        [Range(0.001f, .1f)]
+        [Tooltip("Distance behind GridObjectCollection")]
+        private float Distance = 0.001f;
```

Comment:
Also a nit on naming these private fields: we've tended to use `camelCase` for private fields.